Feature/meeting entity

taejunUM opened the pull request:
## 📄 feat: Meeting entity & repository 구현
<!-- ex. feat: 회원가입 API 구현 -->

#### 📎 관련 이슈
<!-- 연결된 이슈 번호 -->
close #11 

## ✅ 변경 사항
<!-- 어떤 작업을 했는지 간략히 정리 -->
- Meeting Entity 구현
- Meeting JPA Repository 구현
- ClubMember 미구현으로 주석처리 (TODO)


## 🔍 테스트 방법
<!-- 어떤 방식으로 동작을 검증했는지 -->
- [x] 로컬에서 직접 테스트
- [ ] 유닛 테스트 작성 및 통과
- [ ] 브라우저 / 앱 화면에서 확인

taejunUM committed:
feat: Meeting Entity 구현

taejunUM committed:
feat: Meeting Repository 구현

# 본문은 아래에 작성

# 꼬릿말은 아래에 작성: ex) Github issue #23

# --- COMMIT END ---
# feat	    새로운 기능 추가
# fix	    버그 수정
# docs	    문서 변경 (README 등 문서 파일 수정)
# style	    코드 스타일 변경 (세미콜론, 들여쓰기, 공백 등)
# refactor  코드 리팩터링 (기능 변화 없음, 변수명 정리 등)
# test	    테스트 코드 추가 및 수정
# chore	    빌드 설정, 패키지 매니저 설정, 기타 중요하지 않은 작업
# ------------------
#   커밋 메시지는 영문 소문자로 시작
#   형식은 type: message 사용
#   하나의 커밋에는 하나의 목적만 작성
#   메시지는 명확하게 무엇을 했는지 설명
# ------------------
#   <Optional>
#   하위 커밋 설명이 필요한 경우, 본문(body)을 줄바꿈하여 추가 설명 작성
#   feat: support social login
#
#   Added Kakao and Google login support.
#   Refactored auth controller to handle provider-based login.

taejunUM committed:
feat: Meeting Entity 정적팩토리메서드 구현

# 본문은 아래에 작성
커스텀생성자, 정적팩토리 메서드 추가
# 꼬릿말은 아래에 작성: ex) Github issue #23

# --- COMMIT END ---
# feat	    새로운 기능 추가
# fix	    버그 수정
# docs	    문서 변경 (README 등 문서 파일 수정)
# style	    코드 스타일 변경 (세미콜론, 들여쓰기, 공백 등)
# refactor  코드 리팩터링 (기능 변화 없음, 변수명 정리 등)
# test	    테스트 코드 추가 및 수정
# chore	    빌드 설정, 패키지 매니저 설정, 기타 중요하지 않은 작업
# ------------------
#   커밋 메시지는 영문 소문자로 시작
#   형식은 type: message 사용
#   하나의 커밋에는 하나의 목적만 작성
#   메시지는 명확하게 무엇을 했는지 설명
# ------------------
#   <Optional>
#   하위 커밋 설명이 필요한 경우, 본문(body)을 줄바꿈하여 추가 설명 작성
#   feat: support social login
#
#   Added Kakao and Google login support.
#   Refactored auth controller to handle provider-based login.

Developer-Groo commented on src/main/java/com/mobble/mobbleserver/meeting/entity/Meeting.java:
```diff
@@ -0,0 +1,70 @@
+
+    //TODO ClubMember 필요
+    @Builder
+    public Meeting(/*ClubMember clubMember, */ String title, LocalDateTime datetime, String location, String cost, int limit, MeetingType type) {
```
일반 생성자는 private 으로 잠궈 주세요

Developer-Groo commented on src/main/java/com/mobble/mobbleserver/meeting/entity/MeetingType.java:
```diff
@@ -0,0 +1,15 @@
+
+    private final String displayName;
+
+    MeetingType(String displayName) {
```
생성자는 `@RequiredArgsConstructor` 적용해주세요

Developer-Groo commented on src/main/java/com/mobble/mobbleserver/meeting/entity/Meeting.java:
```diff
@@ -0,0 +1,70 @@
+    @Enumerated(EnumType.STRING)
+    private MeetingType type;
+
+
```
공백은 한칸으로 수정해주세요

taejunUM committed:
refactor: meeting entity 수정

Meeting
- 생성자 접근제어자 public > private
- 공백 수정

MeetingType
- 생성자 @RequiredArgsConstructor 대체
